libp2p/src/builder/phase/bandwidth_logging.rs

```diff
@@ -13,28 +11,6 @@ pub struct BandwidthLoggingPhase<T, R> {
 impl<T: AuthenticatedMultiplexedTransport, Provider, R>
     SwarmBuilder<Provider, BandwidthLoggingPhase<T, R>>
 {
-    #[allow(deprecated)]
-    #[deprecated(note = "Use `with_bandwidth_metrics` instead.")]
-    pub fn with_bandwidth_logging(
-        self,
-    ) -> (
-        SwarmBuilder<Provider, BandwidthMetricsPhase<impl AuthenticatedMultiplexedTransport, R>>,
-        Arc<BandwidthSinks>,
-    ) {
-        let (transport, sinks) = self.phase.transport.with_bandwidth_logging();
-        (
-            SwarmBuilder {
-                phase: BandwidthMetricsPhase {
-                    relay_behaviour: self.phase.relay_behaviour,
-                    transport,
-                },
-                keypair: self.keypair,
-                phantom: PhantomData,
-            },
-            sinks,
-        )
-    }
-
     pub fn without_bandwidth_logging(self) -> SwarmBuilder<Provider, BandwidthMetricsPhase<T, R>> {
```

Comment:
Just had a look at this PR again. Without the `with_bandwidth_logging` option the whole `BandwidthLoggingPhase` doesn't make any sense anymore. We should remove this whole module and in the code that formerly transitioned to `BandwidthLoggingPhase` instead directly transition to `BandwidthMetricsPhase`.

Reply:
cc @jxs

Reply:
yeah, let's go for it! 
Thanks Elena for bringing this and DrHuang for the ping